addons/website_hr_recruitment/data/hr_job_demo.xml

```diff
@@ -156,17 +156,13 @@
                             <div>
```

Comment:
@fja-odoo You mention a commit of odoo-dev in your commit message, mention the one from odoo.